First pass at L1Trigger Layer1 DQM module 81X

nsmith- opened the pull request:
For integration into production DQM sequence

Cherry-picked from 8_0_1, see #13733 
Conflicts:
    DQM/Integration/python/clients/l1tstage2_dqm_sourceclient-live_cfg.py
    DQM/L1TMonitor/python/L1TStage2_cff.py

nsmith- committed:
First pass at L1Trigger Layer1 DQM module

For integration into production DQM sequence

Cherry-picked from 8_0_1
Conflicts:
	DQM/Integration/python/clients/l1tstage2_dqm_sourceclient-live_cfg.py
	DQM/L1TMonitor/python/L1TStage2_cff.py

cmsbuild commented:
A new Pull Request was created by  @nsmith- (Nicholas Smith) for CMSSW_8_1_X.

It involves the following packages:

DQM/Integration
DQM/L1TMonitor


@cmsbuild, @vanbesien, @deguio, @davidlange6 can you please review it and eventually sign? Thanks.
@threus, @batinkov this is something you requested to watch as well.
@slava77, @degano, @smuzaffar you are the release manager for this.

cms-bot commands are list here cms-sw/cmssw#13028

nsmith- commented:
Oh, looks like someone finally fixed the `DQMOffline/L1Trigger` BuildFile issue in #13691
That causes the conflict

nsmith- committed:
CaloLayer1 DQM bugfix

TPs start at 1!

cmsbuild commented:
Pull request #13734 was updated. @cmsbuild, @vanbesien, @deguio, @davidlange6 can you please check and sign again.

nsmith- committed:
Calo Layer1 update: remove separate received HF digi

HF and HCAL digis will be combined in L1CaloLayer1 unpacker in this:
https://github.com/cms-sw/cmssw/pull/13723
This branch would then cause an error if we include HF digis as
currently implemented.

cmsbuild commented:
Pull request #13734 was updated. @cmsbuild, @vanbesien, @deguio, @davidlange6 can you please check and sign again.

deguio commented:
-1
doesn't merge.

nsmith- commented:
Included in #13800
